assets/src/blocks/Covers/CoversEditor.js

```diff
@@ -47,12 +47,15 @@ const renderEdit = (attributes, toAttribute, setAttributes) => {
       </PanelBody>
       <PanelBody title={__('Settings', 'planet4-blocks-backend')}>
         {layout !== COVERS_LAYOUTS.carousel &&
-          <SelectControl
-            label="Rows to display"
-            value={initialRowsLimit}
-            options={rowLimitOptions}
-            onChange={value => toAttribute('initialRowsLimit')(Number(value))}
-          />
+          <>
+            <label htmlFor="row-amount-select">{__('Rows to display', 'planet4-blocks-backend')}</label>
```

Comment:
This needed to be updated because otherwise WordPress would add a super specific class name to the label, that was overriding our own overrides 😅